azure-pipelines.yml

```diff
@@ -147,9 +147,28 @@ stages:
           displayName: Build
           condition: succeeded()
 
-- ${{ if and(ne(variables['System.TeamProject'], 'public'), notin(variables['Build.Reason'], 'PullRequest')) }}:
```

Comment:
Do we need to run policheck based on this condition or always? I have removed this since arcade definition does not have it. 

Adding @wtgodbe  as well.

Reply:
ASP.NET tooling repo still has this check.https://github.com/dotnet/aspnetcore-tooling/blob/783bfb534322a8a344d13c5c08d2d3b742ecf365/azure-pipelines.yml#L372

Reply:
I think it's required.

Reply:
Added back